a10_neutron_lbaas/a10_context.py

```diff
@@ -108,6 +111,19 @@ def __exit__(self, exc_type, exc_value, traceback):
         super(A10WriteContext, self).__exit__(exc_type, exc_value, traceback)
 
 
+class A10ReplayContext(A10WriteContext):
+
+    def __init__(self, *args, **kwargs):
+        self._appliance = kwargs.pop('appliance', None)
```

Comment:
No, not vThunder.  It's a "replay context", so it's state variable is probably referring to "the thing i want to replay against". Which should have a few expected interfaces.  replay_entity? host_collection?  Ideally whatever it's called is the same as the base class that defines the interface this context expects to interact with.